mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogPresenter.java

```diff
@@ -86,13 +100,25 @@ public void onCompleted() {
                     @Override
                     public void onError(Throwable e) {
                         getMvpView().showProgressbar(false);
-                        getMvpView().showError(R.string.failed_to_update_document);
+                        if (e instanceof HttpException) {
```

Comment:
@tarun0 why don't you use this https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/utils/MFErrorParser.java#L25